### PR TITLE
Add the config module functionality to tests

### LINKED ISSUE #18: Replace configuration code with new config module
build-artifacts.py and system-tests.py duplicate configuration code. They should use the new config modules added in #16 

- https://github.com/nirs/oci/blob/857f949d17dcd6b3cd985abe1b55aa3c6444affc/build-artifacts.py#L18
- https://github.com/nirs/oci/blob/857f949d17dcd6b3cd985abe1b55aa3c6444affc/system-tests.py#L25

@Funkabell do you want to take this?

### PR DESCRIPTION
**Pull request by bell-lev**
After adding a module to parse config files (#16), adding this
functionality to build-artifacts and system-tests test runs.

Resolves #18

### CODE REVIEW

File: build-artifacts.py [hunk 1]
```diff
@@ -15,16 +13,14 @@
 
 change = sys.argv[1]
 
-config = configparser.ConfigParser()
-conf_path =  os.path.expanduser("~/.config/oci.conf")
-config.read(conf_path)
+cfg = config.load("~/.config/oci.conf")
```
Comment (nirs) [inline build-artifacts.py]: I think we should let config module load the default file. The only reason
to use special path is when a user ask to do this.

File: system-tests.py [hunk 2]
```diff
@@ -22,16 +20,14 @@
 
 suite_type = "basic"
 
-config = configparser.ConfigParser()
-conf_path =  os.path.expanduser("~/.config/oci.conf")
-config.read(conf_path)
+cfg = config.load("~/.config/oci.conf")
```
Comment (nirs) [inline system-tests.py:23]: Same, use default.

**Review by nirs — CHANGES_REQUESTED** [review]
Nice, but lets use the default config path.

**Comment by nirs** [comment]
You can also rebase on master so the tests will pass.

**Comment by bell-lev** [comment]
> You can also rebase on master so the tests will pass.

right, will do.

**Comment by nirs** [comment]
Thanks Bell!

**Pull request merged**